Configuring Che installation on GCP (#1603)
codenvy/che-docs: modules/installation-guide/partials/proc_configuring-the-che-operator-checluster-resource-with-kubectl.adoc

[id="configuring-the-che-operator-checluster-resource-with-kubectl_{context}"]

> = Configuring CheCluster Custom Resource with kubectl

Reviewer: ```suggestion
= Configuring CheCluster Custom Resource with {prod-cli}
```

> Edit the CheCluster custom resource configuration with kubectl:

Reviewer: ```suggestion
Edit the CheCluster custom resource configuration with {prod-cli}:
```

> 

Reviewer: ```suggestion
+
[subs="+attributes"]
```

----

> kubectl edit CheCluster/eclipse-che -n che

Reviewer: ```suggestion
$ {prod-cli} edit CheCluster/eclipse-che -n che
```
